.github/contributing/pull_requests.md

```diff
@@ -0,0 +1,224 @@
+There are plenty of examples to draw from in the existing resources, but you still get to implement something completely new.
+
+In addition to the below checklist, please see the [Common Review Items](#common-review-items) sections for more specific coding and testing guidelines.
+
+ - [ ] __Minimal LOC__: It's difficult for both the reviewer and author to go through long feedback cycles on a big PR with many resources.
+   We ask you to only submit **1 resource at a time**.
+ - [ ] __Acceptance tests__: New resources should include acceptance tests covering their behavior.
+   See [Writing Acceptance Tests](#writing-acceptance-tests) below for a detailed guide on how to approach these.
+ - [ ] __Resource Naming__: Resources should be named `scaleway_<service>_<name>`, using underscores (`_`) as the separator.
+   Resources are namespaced with the service name to allow easier searching of related resources, to align the resource naming with the service for [Customizing Endpoints](https://www.terraform.io/docs/providers/scaleway/guides/custom-service-endpoints.html#available-endpoint-customizations), and to prevent future conflicts with new Scaleway services/resources.
+   For reference:
+
+   - `service` is the Scaleway short service name that matches the entry in `endpointServiceNames` (created via the [New Service](#new-service) section)
+   - `name` represents the conceptual infrastructure represented by the create, read, update, and delete methods of the service API.
+     It should be a singular noun.
+     For example, in an API that has methods such as `CreateThing`, `DeleteThing`, `DescribeThing`, and `ModifyThing` the name of the resource would end in `_thing`.
+
+ - [ ] __Arguments_and_Attributes__: The HCL for arguments and attributes should mimic the types and structs presented by the Scaleway API.
+   API arguments should be converted from `CamelCase` to `camel_case`.
+ - [ ] __Documentation__: Each resource gets a page in the Terraform documentation.
+   The [Terraform website][website] source is in this repo and includes instructions for getting a local copy of the site up and running if you'd like to preview your changes.
+   For a resource, you'll want to add a new file in the appropriate place and add a link to the sidebar for that page.
+ - [ ] __Well-formed Code__: Do your best to follow existing conventions you see in the codebase, and ensure your code is formatted with `go fmt`.
+   (The Travis CI build will fail if `go fmt` has not been run on incoming code.)
+   The PR reviewers can help out on this front, and may provide comments with suggestions on how to improve the code.
+ - [ ] __Vendor updates__: Create a separate PR if you are adding to the vendor folder.
+   This is to avoid conflicts as the vendor versions tend to be fast-moving targets.
+   We will plan to merge the PR with this change first.
+
+#### New Service
```

Comment:
This whole section does not apply as adding a new service only consist of addin a ressource in this service